rld/test/phdr.json

```diff
@@ -32,14 +32,13 @@
         {
             // generation 1
```

Comment:
Should change `generation` to `transaction`?

Reply:
IMO there’s no real need.